tests/rados/test_fast_ec_config_params.py

```diff
@@ -935,6 +944,659 @@ def run(ceph_cluster, **kw):
+
+            # Print EC pool and profile details for pool_name_1
+            print_ec_pool_profile_details(rados_obj, pool_name_1)
+
+            # Verify pool_name_1 has EC optimizations disabled
+            ec_opt_value_dict = rados_obj.get_pool_property(
+                pool=pool_name_1, props="allow_ec_optimizations"
+            )
+            ec_opt_value = ec_opt_value_dict.get("allow_ec_optimizations")
+            log.info("Pool %s - allow_ec_optimizations: %s", pool_name_1, ec_opt_value)
+            if ec_opt_value is not False:
+                log.error("allow_ec_optimizations is not False for %s", pool_name_1)
+                raise Exception("EC optimizations check failed for %s" % pool_name_1)
+
+            # Step 3: Create second EC pool with 16K stripe_unit and enable optimizations
+            log.info(
+                "Step 3: Creating EC pool WITH optimizations: %s (stripe_unit=%s)",
+                pool_name_2,
+                stripe_unit,
+            )
+            assert rados_obj.create_erasure_pool(
+                pool_name=pool_name_2,
+                profile_name=f"ec_profile_{pool_name_2}",
+                k=2,
+                m=2,
+                stripe_unit=stripe_unit,
+                app_name="rados",
+            ), f"Failed to create EC pool {pool_name_2}"
+
+            rados_obj.set_pool_property(
+                pool=pool_name_2, props="allow_ec_optimizations", value="true"
+            )
+            log.info("Enabled allow_ec_optimizations on pool %s", pool_name_2)
+
+            time.sleep(2)
+
+            # Print EC pool and profile details for pool_name_2
+            print_ec_pool_profile_details(rados_obj, pool_name_2)
+
+            # Verify pool_name_2 has EC optimizations enabled
+            ec_opt_value_dict = rados_obj.get_pool_property(
+                pool=pool_name_2, props="allow_ec_optimizations"
+            )
+            ec_opt_value = ec_opt_value_dict.get("allow_ec_optimizations")
+            log.info("Pool %s - allow_ec_optimizations: %s", pool_name_2, ec_opt_value)
+            if ec_opt_value is not True:
+                log.error("allow_ec_optimizations is not True for %s", pool_name_2)
+                raise Exception("EC optimizations check failed for %s" % pool_name_2)
+
+            # Step 4: Write objects smaller than stripe width (16KB) to both pools
+            # Writing 4KB objects to demonstrate zero-padding difference
+            # Note: We use 'rados put' instead of 'rados bench' because rados bench
+            # rounds up object sizes to stripe_width for EC pools.
+            # Without optimizations: 4KB padded to 16KB stripe_unit = 4x storage overhead
+            # With optimizations: 4KB stored as-is = normal 2x EC overhead
+            log.info(
+                "Step 4: Writing objects (4KB each, smaller than 16KB stripe width) to both pools"
+            )
+            num_objects = 100
```

Comment:
In the test steps mentioned that writing 200  identical 4KB objects but num_objects are 100.